spec/mailers/previews/reservation_preview.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Do you need this file?

Reply:
No, I don't. I use letter opener, so don't need this.